Lecture2/Task1/Task1L2.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Scanner;
+
+public class Task1L2 {
+    public static void main ( String[] args ) {
+        Scanner in = new Scanner ( System.in );
+        System.out.print ( "Please enter a sentence: " );
+        String sentence = in.nextLine ();
+        String reverse = new StringBuffer(sentence).reverse().toString();
```

Comment:
good choice :) However i would like you to do it in another way, without using java internal mechanism.

Reply:
fixed